lib/gui/app/scss/modules/_theme.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"),
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+exports.colors = {
+  $paletteThemeDarkForeground: '#fff',
```

Comment:
Why are all these colors prefixed with `$paletteTheme`?

Reply:
IIRC `$` indicates a variable in Sass and Sass does not support namespacing. Since the theme module is nicely namespaced this file could be reformatted so that you access colors like
```
exports.colors = {
  dark: {
    foreground: '#fff'
    background: '#4d5057'
  }
  // etc...
}
const color = them.colors.dark.foreground;
```